### ISSUE TITLE
wiki: Configuration ... 'export' confusing.

### ISSUE THREAD

**Issue by bs27975** [astrada/google-drive-ocamlfuse#205]
Reading https://github.com/astrada/google-drive-ocamlfuse/wiki/Configuration I am confused by the term 'export' - I am looking to mirror the files exactly as on gdrive, not export (==translate?) anything.
(Mostly as a backup for when a shared folder user inadvertently deletes / overwrites  a file / version.)

Could the wiki page be updated to further clarify the term 'export'? e.g. Does it mean translate? e.g. Retrieve a gsheets or xls file as ods?

Thus, correspondingly, one should leave all _format= lines blank?

Also, I appreciate the "The parser is very simple and only accepts lines in the form key=value, without spaces, comments, or anything else." comment. A comment line, prefixed with '#', would be appreciated. e.g. When first starting things up, figuring things out, and not wanting to lose whatever an original setting was / insert a comment/note regarding the presence / why of a change.

If by chance prefixing a line with # is indeed ignored, or not, a note to that effect, in the wiki page would be useful.

Thanks for listening.

**Comment by bs27975** [comment]
Should the config defaults not be (include):
document_format=gdoc
drawing_format=gdraw
form_format=gform
map_format=gmap
presentation_format=gslides
spreadsheet_format=gsheet

**Comment by astrada** [comment]
No, Google Docs cannot be downloaded in native format. Your only option is to export them to one of the supported format (https://developers.google.com/drive/v3/web/manage-downloads#downloading_google_documents).

**Comment by bs27975** [comment]
Thanks. I was all set to close this git issue, then - except, it does seem pointful to update the wiki article to make explicit that export == translate - and a Google limitation beyond anyone's control. (Which may be self-evident to those with such experience, but to those new to things, such as the likely readers of that wiki page, not so much.) This git issue could then be closed.
